driver/src/bin/models.rs

```diff
@@ -14,9 +14,9 @@ pub struct State {
 #[derive(Serialize, Deserialize)]
 pub struct Deposits {
   	pub depositHash: String,
-    depositIndex: String,
-    slot: i32,
-    addressId: i32,
-    tokenId: i32,
-    amount: i32,
+    pub depositIndex: String,
```

Comment:
This value equivalent to slot and not used.

Reply:
No, we agreed to have a counter per slot to sort them in order, didn't we?

How else would I know which one is the latest deposit with the correct depositHash? 
@bh2smith

Reply:
Interesting. Yes, this definitely matters. I will make this happen. Whoops.

Reply:
In that case, I think this depositIndex will have to be an `Integer`